### DISCUSSION TITLE
Dated folders for image generation

### DISCUSSION THREAD

**Discussion by fiveseven808** [Kilvoctu/aiyabot#160] [Ideas]
My bot instance has created 3.7k images since 3/19 and I was wondering if we could get images in the output folder organized into folders by date? it would help sorting and looking for specific images quite a bit.

**Comment by Kilvoctu** [comment]
Heya, I know how it feels. I have thousands of images, too, and I sort them into folders. I do this with pretty much any files on my PC, like game screenshots, music, and such. I love organization 👍  

That said, I don't think this is a feature I'll be doing. File management/organization is not something that's within the scope of the project. There are some things you can try to help with sorting your files that I thought of:  

- You can use the [Image Browser extension](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser) for Web UI and point it at your outputs folder. The extension is a bit cumbersome, but it works to both search images and sort them into folders. This is what I use when I'm on my mobile device.

- [Diffusion Toolkit](https://github.com/RupertAvery/DiffusionToolkit) is something I tried for a bit in the past, which is nice for looking for specific images. However, I don't think it has any sorting features. Nevermind I checked just now and [it does seem possible](https://github.com/RupertAvery/DiffusionToolkit/issues/56), so this is probably better than the above extension. Guess I'll be using this when  at my PC.
 
- You can use a file sorting application. For example, [RoboBasket](https://www.robobasket.com/index.html). I've tried this and it can do [exactly what you're asking for](https://www.robobasket.com/auto-sort-files-into-folders.html#sortbydate) very easily and automatically. I used it just now to sort my images in outputs into dated folders; it works. It is a paid program, however, though there is a free trial so you can try it out.
